GenerateReport.py

```
@@ -388,14 +402,42 @@ def main() -> None:
+        and 'not used' not in r['Remarks'].lower()
+    )
+    dep_vuln_notused = sum(
+        1 for r in rows
+        if r['Package Type'] == 'Dependency Package'
+        and r['Current Version Vulnerable?'] == 'Yes'
+        and 'not used' in r['Remarks'].lower()
+    )
 
     logger.info("📦 Weekly Report Summary")
     logger.info(f"🔍 Total packages scanned: {total} (Base: {base_count}, Dependency: {dep_count})")
     logger.info(f"🚨 Vulnerabilities found in current versions:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)